cmd/selectionMLE/tmp.txt

```diff
@@ -0,0 +1,2 @@
+#FILENAME	MaximumLikelihood
```

Comment:
Should this be in testdata?

Reply:
I commented out the os.Remove while debugging, will fix on next commit